policy/LAM/integration_test.yaml

```diff
@@ -0,0 +1,57 @@
+#
+# Known calib types
+#
+calibTypes:
+  [
+  bias, dark, flat, fiberTrace, arc
+  ]
+#
+# Pipeline steps
+#
+scienceSteps:
+  [
+  reduceExposure, mergeArms, calculateReferenceFlux, fluxCalibrate, coaddSpectra
+  ]
+#
+# Bootstrap the calibration directory
+#
+# The files are to be found in dirName, taken to be relative to --dataDir if
+# not an absolute path
+#
+bootstrap :
+  dirName : "$DRP_STELLA_DATA_DIR/raw"
+  arms : [r1, b1]
+  detectorMapFmt : detectorMap-sim-%s.fits
+#
+# Blocks of data which are used together to update the calibrations
+#
+calibBlocks :
+  integration_test :
+    bias :
+      visits : ["0..4"]
```

Comment:
Why is this a list? Shouldn't it just be a string? Or do different elements in the list result in different executions of `constructBias`?

Reply:
Originally, this had to be a list because of `visit` and `visits` distinction. Now these two keys has been merged to `visit`, and this value does not have to be a list.